### PR TITLE
Add Truly Personal to composer.json

### PR DESCRIPTION
**Pull request by RCheesley**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | N/A
| Related developer documentation PR URL | N/A
| Issue(s) addressed                     | N/A

## Description

This PR adds the new Simply Personal theme to the composer.json file so that it is picked up for composer-based installations - it's a rebase of #10134 for the 5.1 branch (sorry it took so long 😊 )

---
### 📋 Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Check that the Simply Personal theme is installed and available to use.

### CODE REVIEW

**Review by shinde-rahul — APPROVED** [review]
~~Code changes look good to me.~~

File: composer.lock
Comment (shinde-rahul) [inline composer.lock:1]: We should exclude this lock file as the theme `trulypersonal` is already part of the code.

**Review by shinde-rahul — CHANGES_REQUESTED** [review]

**Comment by matbcvo** [comment]
This PR can be closed because the theme has already been added to the `composer.json` file through PR #14301.

https://github.com/mautic/mautic/blob/b121176632d980cbd388c14d51ed8dbe8bb02aa4/composer.json#L100

**Comment by escopecz** [comment]
@RCheesley based on the learnings in https://github.com/mautic/mautic/pull/14301 this PR should just update the composer.lock hash, not adding the theme in. Can you please update composer.lock with just the hash?

**Comment by RCheesley** [comment]
@escopecz this PR can be closed now that it's added: https://github.com/mautic/mautic/blob/5.x/composer.json#L100

**Pull request closed without merging**